scripts/run-cypress

```diff
@@ -6,5 +6,8 @@ if ! [ -x "$(command -v cypress)" ]; then
   exit 1
 fi 
 
+echo "Setting up test runner"
+node dist/testRunner.js
```

Comment:
This process won't terminate as the server should listen indefinitely. This should mean that `cypress run` is never actually reached. 

I don't have a suggestion right away but we'll aim for this to make sure the server is running and run in parallel.

Reply:
Closing this as another PR is resolving this issue.